packaging/centreon-agent.yaml

```diff
@@ -0,0 +1,66 @@
+name: "centreon-agent"
+arch: "${ARCH}"
+platform: "linux"
+version_schema: "none"
+version: "${VERSION}"
+release: "${RELEASE}${DIST}"
+section: "default"
+priority: "optional"
+maintainer: "Centreon <contact@centreon.com>"
+description: |
+  This software is an agent used to executes commands on remote computers as nsclient does.
+  Commit: @COMMIT_HASH@
+vendor: "Centreon"
+homepage: "https://www.centreon.com"
+license: "Apache-2.0"
+
+contents:
+  - src: "../agent/conf/centagent.json"
+    dst: "/etc/centreon-agent/centagent.json"
+    type: config|noreplace
+    file_info:
+      mode: 0664
+      owner: centreon-agent
+      group: centreon-agent
+
+  - src: "../agent/scripts/centagent.service"
+    dst: "/usr/lib/systemd/system/centagent.service"
+    file_info:
+      mode: 0644
+    packager: rpm
+  - src: "../agent/scripts/centagent.service"
+    dst: "/lib/systemd/system/centagent.service"
+    file_info:
+      mode: 0644
+    packager: deb
+
+  - src: "../build/agent/centagent"
+    dst: "/usr/bin/centagent"
+
+  - dst: "/etc/centreon-agent"
```

Comment:
This will not do well, as the path, user and group are already owned and used by another piece of software from centreon and I am not sure this can be easily changed. As they will overlap, it may lead to issues regarding configuration and permissions for them.